src/scipp/core/binning.py

```diff
@@ -60,8 +60,8 @@ def make_histogrammed(x, *, edges):
         x = DataArray(data, coords={edges.dim: x})
     elif isinstance(x, DataArray) and x.bins is not None:
         dim = edges.dims[-1]
-        if dim not in x.bins.meta:
-            if x.meta.is_edges(dim):
+        if dim not in x.bins.coords:
```

Comment:
This is a breaking change that removes some support for attrs. Should we actually do this now?

Reply:
No you did not misunderstand, but didn't we also change, e.g., `transform_coords` to not look for `attrs`?

Reply:
Yes, looks like it. But that was when we moved to aligned coords, not when we deprecated attrs. I'd actually say we should change it along with the code here to use `deprecated_meta`.

Reply:
> But that was when we moved to aligned coords, not when we deprecated attrs. 

Indeed, should we change it in another PR?

> we should change it along with the code here to use deprecated_meta.

But when should we make the breaking change then? Why not now, if the similar changes have already happened recently?

Reply:
> But when should we make the breaking change then? Why not now, if the similar changes have already happened recently?

I would have thought we do this when removing attrs. The earlier changes may have been a mistake and we shouldn't use that to continue down the same path.

But maybe the whole point is moot because nobody uses attrs like this?

Reply:
This is unrelated to removal of attrs, isn't it? This is about the addition of the coord alignment flag. The "only" reason all those operations also considered attrs was their double-use for unaligned coords.

> The earlier changes may have been a mistake

In what sense? Are you considering going back to attrs instead of unaligned coords?

Reply:
So do you see the introduction of unaligned coords also as removing that role from attrs?

Reply:
Yes